src/phpDocumentor/Descriptor/Property.php

```diff
@@ -0,0 +1,129 @@
+<?php
+/**
+ * phpDocumentor
+ *
+ * PHP Version 5.3
+ *
+ * @copyright 2010-2014 Mike van Riel / Naenius (http://www.naenius.com)
+ * @license   http://www.opensource.org/licenses/mit-license.php MIT
+ * @link      http://phpdoc.org
+ */
+
+namespace phpDocumentor\Descriptor;
+
+use phpDocumentor\Reflection\DocBlock;
+use phpDocumentor\Reflection\Element;
+use phpDocumentor\Reflection\Fqsen;
+use phpDocumentor\Reflection\Php\Visibility;
+
+/**
+ * Descriptor representing a property.
+ */
+final class Property implements Element
+{
+    /**
+     * @var Fqsen
+     */
+    private $fqsen;
+
+    /**
+     * @var DocBlock
+     */
+    private $docBlock;
+
+    /** @var string[] $types */
+    protected $types = array();
+
+    /** @var string $default */
+    protected $default = array();
+
+    /** @var bool $static */
+    protected $static = false;
+
+    /** @var string $visibility */
+    protected $visibility = 'public';
```

Comment:
should be private and not default to a string but to null. Also: the DocBlock should mention the Visibility class